Makefile

```diff
@@ -29,15 +29,15 @@ build: build.bin build.docker ## All-in-one build
 
 .PHONY: build.bin
 build.bin: export CGO_ENABLED = 0
-build.bin: fmt vet templates/bootstrap.min.css ## Build binary
+build.bin: generate fmt vet templates/bootstrap.min.css ## Build binary
```

Comment:
probably better add `templates/bootstrap.min.css` also to `vet` (and if necessary, `fmt`) targets